### DISCUSSION TITLE
Voice control from Google Assistant to Music Assistant

### DISCUSSION THREAD

**Discussion by dev77-07** [music-assistant/support#3307] [Q&A] [answered]
I'm sorry if this has been answered before but I haven't came across it.

I have a few different Google Assistant devices (Gen. 1 Google Home, some Lenovo smart clocks) around the house. I can play to them using the Music Assistant UI since they show up as Google Cast devices for playback. What I'm hoping to do is use the Google Assistant to say "Play Metallica on Bedroom Speaker".

Looking at the MA documentation I see steps to do this through HA Assist but can't figure out how to bridge Google Assistant, HA Assist and Music Assistant. I do have Home Assistant set up and I have the Google Assistant integration working along with Music Assistant integration. 

Is there any way to link them all together somehow to allow me to start playing music from Music Assistant using Google Assistant?

**Comment by OzGav** [comment, marked as answer]
No that is not currently possible.

**Reply by jonasjeeliasson** [reply]
Sry to drag up something old. Is this something that exists in the backlog. Would love to have Music Assistant as my default music provider instead of Spotify in Google Home for example.

**Reply by OzGav** [reply]
Noits not  because I expect that would require a whole lot of pain to get into "Works with Google Assistant"

**Comment by homeauto-git** [comment]
I used Brave and from what I read its possible but you need to be a google developer to do it. Your basically trying to pass google assistant intents to home assistant and then process automation from there. I did not do too much digging into it but here some links I think might be relevant.
https://developers.google.com/assistant/conversational/intents
https://www.home-assistant.io/integrations/conversation/

I also think you could probably do it with ifttt.

https://support.google.com/googlenest/answer/7194656?hl=en&co=GENIE.Platform%3DDesktop
